Set1Task6/Program.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+
+namespace Set1Task6
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Random dice = new Random();
+            int player1 = dice.Next(1,7);
+            int player2 = dice.Next(1,7);
+            int player3 = dice.Next(1, 7);
+            int player4 = dice.Next(1, 7);
+            int player5 = dice.Next(1, 7);
+
+            Console.WriteLine("Player 1: " + player1);
+            Console.WriteLine("Player 2: " + player2);
+            Console.WriteLine("Player 3: " + player3);
+            Console.WriteLine("Player 4: " + player4);
+            Console.WriteLine("Player 5: " + player5);
+
+            if ((player1 == player2)|| (player1 == player3)|| (player1 == player4)|| (player1 == player5))
```

Comment:
player1 == player2 is same as player2 == player1.
In this case both player1 and player2 are moved to next level